code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -411,6 +412,9 @@
 				affected_mob.Unconscious(12 SECONDS)
 				to_chat(affected_mob, span_cult_large("[pick("Your blood is your bond - you are nothing without it", "Do not forget your place", \
 					"All that power, and you still fail?", "If you cannot scour this poison, I shall scour your meager life!")]."))
+		else if(HAS_TRAIT(affected_mob, TRAIT_EVIL) && SPT_PROB(25, seconds_per_tick)) //Congratulations, your committment to evil has now made holy water a deadly poison to you!
+			affected_mob.emote("scream")
+			need_mob_update += affected_mob.adjustBruteLoss(3 * REM * seconds_per_tick, updating_health = FALSE)
```

Comment:
should be burn damage, no?

Reply:
same with the other place they take brute damage lower down

Reply:
oh you're right, it should be